pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.24</dep.slice.version>
         <dep.aws-sdk.version>1.9.40</dep.aws-sdk.version>
-        <dep.tempto.version>1.12</dep.tempto.version>
+        <dep.tempto.version>1.15-SNAPSHOT</dep.tempto.version>
```

Comment:
Would this be temporary until we do an actual release of Tempto, when you would change it to a release version (i.e. one without the `SNAPSHOT` tag) ?